src/herder/LedgerCloseData.cpp

```diff
@@ -14,9 +14,10 @@ using namespace std;
 namespace stellar
 {
 
-LedgerCloseData::LedgerCloseData(
-    uint32_t ledgerSeq, std::shared_ptr<AbstractTxSetFrameForApply> txSet,
-    StellarValue const& v, std::optional<Hash> const& expectedLedgerHash)
+LedgerCloseData::LedgerCloseData(uint32_t ledgerSeq,
+                                 std::shared_ptr<TxSetFrame> txSet,
```

Comment:
nit: We have `TxSetFramePtr`, which looks a bit more concise (and also consistent with `TxFramePtr`). Could you please update the relevant places to use that alias instead?